mmdet3d/models/losses/monoflex_ious_loss.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch import nn as nn
+
+from mmdet.models.builder import LOSSES
+from mmdet.models.losses.utils import weighted_loss
+
+
+@weighted_loss
+def monoflex_iou_loss(pred, target, mode='iou', eps=1e-7):
+    """FCOS-style IoU loss.
```

Comment:
Can we use the iou loss in the mmdet?

Reply:
If we want to use the iou loss in mmdet,  it may need some conversion before feeding into the loss function